src/main/java/com/exasol/containers/LiteralExasolDockerImageReference.java

```diff
@@ -0,0 +1,72 @@
+package com.exasol.containers;
+
+import java.util.Optional;
+
+/**
+ * Docker image reference for images that don't follow the established standard of naming Exasol Docker images.
+ */
+public class LiteralExasolDockerImageReference implements ExasolDockerImageReference {
```

Comment:
maybe you can make this package-private so that it is clear that it is not a public API but the factory should be used?

Reply:
Done., also for the other variant.